### PR TITLE
docs: add workspace classes description

### PR DESCRIPTION
**Pull request by stefanicjuraj**
Fixes https://github.com/daytonaio/enterprise-docs/issues/94

### CODE REVIEW

File: src/content/docs/usage/workspaces.md [hunk 1]
```diff
@@ -122,11 +122,49 @@ Daytona allows you to delete one or more Workspaces, helping you manage your dev
 
 ## Workspace Classes
 
-The Workspace class allows you to select the amount of resources allocated to your Workspace. You can choose from different configurations depending on your requirements:
+Workspace classes are predefined configurations that allow users to select the amount of resources allocated to their workspace based on their specific requirements. The classes exist to simplify the process of resource allocation, ensuring that users can choose a setup that meets their requirements without manually specifying the details.
+
+Each Workspace class specifies the amount of CPU, RAM, disk storage, and optionally, GPU resources available to a Workspace. The appropriate selection of a Workspace class depends on the intensity and nature of the tasks to be performed within the Workspace.
```
Comment (ivan-burazin) [inline src/content/docs/usage/workspaces.md:128]: @stefanicjuraj these are not fixed workspace classes. Rather the Admin can create any size they want and then the user will be able to chose.

Would suggest you message it in that way that conveys there can be one or more with diff cpu/gpu/hdd/ram depedning on what the Daytona admin definedd

**Review by ivan-burazin — CHANGES_REQUESTED** [review]

File: src/content/docs/usage/workspaces.md [hunk 2]
```diff
@@ -122,11 +122,51 @@ Daytona allows you to delete one or more Workspaces, helping you manage your dev
...
+
+  The Medium class balances performance and resource usage, making it suitable for general-purpose tasks that require moderate processing power and memory.
+
+- **Small**
+
+  The Small class is ideal for light workloads or development environments where minimal computational resources are needed. This class is the most resource-efficient, making it a good choice for simple applications and tasks.
+
+The resources available for each Workspace class are constrained by the hardware node availability within the cluster. Each node in the cluster has a finite amount of CPU, RAM, storage, and GPU resources. The maximum number of Workspaces that can be provisioned depends on these hardware limits.
+
+### Overprovisioning
```
Comment (ivan-burazin) [inline src/content/docs/usage/workspaces.md:156]: This needs to go under Administration > Workspace Classes and should be removed from here

File: src/content/docs/usage/workspaces.md [hunk 3]
```diff
@@ -122,11 +122,51 @@ Daytona allows you to delete one or more Workspaces, helping you manage your dev
...
+The appropriate selection of a Workspace class depends on the intensity and nature of the tasks to be performed within the Workspace.
 
 | **Workspace Class** | **CPU** | **RAM** | **GB** | **GPU** |
 |---------------------|---------|---------|--------|---------|
 | GPU                 | 4       | 16 GB   | 50 GB  | 1       |
 | Large               | 8       | 32 GB   | 50 GB  |         |
 | Medium              | 4       | 16 GB   | 50 GB  |         |
 | Small               | 2       | 8 GB    | 50 GB  |         |
+
+- **GPU**
```
Comment (ivan-burazin) [inline src/content/docs/usage/workspaces.md:138]: @stefanicjuraj all these classes are create arbitrarily so they can be call "john, mark, bob, and stan" so needs to be written in a form that the user understand what are classes not their exact name or specs

**Review by ivan-burazin — CHANGES_REQUESTED** [review]

File: src/content/docs/usage/workspaces.md (hunk 3 above)
Comment (stefanicjuraj) [reply src/content/docs/usage/workspaces.md]: Thanks for the explanation - I have modified the description accordingly.

File: src/content/docs/usage/workspaces.md (hunk 2 above)
Comment (stefanicjuraj) [reply src/content/docs/usage/workspaces.md]: I will create a new pull request for this as there is no administration group created (merged) just yet - the administration group is created as part of the "port user types to administration" pull request [here](https://github.com/daytonaio/enterprise-docs/pull/110/files#diff-e0f0c5adbe0b9ca5d0b57caf5cea33a8d88899fd02a43df1e9862b185f8a1e5f).

**Review by fabjanvucina — APPROVED** [review]

**Pull request merged**